tests/test_registrations/test_archiver.py

```diff
@@ -1130,7 +1130,7 @@ def test_archiving_nodes_not_added_to_search_on_archive_failure(self, mock_send,
                 mock.patch('website.archiver.model.ArchiveJob.success', mock.PropertyMock(return_value=False))
         ) as (mock_finished, mock_sent, mock_success):
             listeners.archive_callback(reg)
-        mock_update_search.assert_not_called()
+        mock_delete_index_node.assert_called()
```

Comment:
I'm not certain that this test is testing anything. `.assert_called` isn't in the [Mock docs](https://docs.python.org/3/library/unittest.mock.html#the-mock-class), and there's a [great blog post](http://engineeringblog.yelp.com/2015/02/assert_called_once-threat-or-menace.html) about why you should be very careful when calling methods on `MagicMocks` that look like they might be asserting things. Excerpt: 

>Here’s a clue: the test will happily continue to pass if you change the assertion to:
>
>`mock_restart.assert_two_wrongs_dont_make_a_right()`
>
>Or:
>
>`mock_restart.assert_two_plus_two_equals(5)`
>
>Or even:
>
>`mock_restart.ASDFASDFASDFASDFASDFASDFASDF()`
>
>What’s going on here?
>
>Remember that a mock’s job is to say, “You got it, boss” whenever anyone calls it.